src/pages/ProgramContentPage/ProgramContentBlock.tsx

```diff
@@ -209,6 +222,12 @@ const ProgramContentBlock: React.VFC<{
               <LockIcon className="mr-2 mb-1" />
               {formatMessage(pageMessages.ProgramContentBlock.loginTrial)}
             </Button>
+          ) : isLoading ? (
+            !showSkeleton && (
+              <div style={{ width: '80%', margin: '0 auto 1.75rem' }}>
+                <SkeletonText mt="4" noOfLines={4} spacing="4" skeletonHeight="2" />
+              </div>
+            )
```

Comment:
loading status should be earlier than the data layer.

Reply:
The loading screen here replaces the original padlock, meaning that the loading screen will appear only when the user needs to wait for verification.